docs/pages/setup/reference/cli.mdx

```diff
@@ -311,6 +311,39 @@ $ tsh play --format=pty 1fe153d1-ce8b-4ef4-9908-6539457ba4ad.tar
 $ tsh play --format=json ~/play/0c0b81ed-91a9-4a2a-8d7c-7495891a6ca0.tar | jq '.event
 ```
 
+### tsh proxy app
+
+Start local TLS proxy for app connections when using Teleport in single-port mode
```

Comment:
What happens if Teleport is not running in single-port mode when you run `tsh proxy app`?

Reply:
I also think there should be a sentence about why you would use this command vs. not using a local proxy for Application Access.

Reply:
This command, as well as the other proxy subcommands seem to be a matter of convenience when working locally. Should I include a note on this?

Reply:
The application works even not in single-port mode due to https being on 443. From my understanding reading the original PR for ALPN Proxy. The `proxy db` and `proxy ssh` will fall back on original connections when teleport is not running in single-port mode. I will attempt to confirm this. The original documentation for the existing commands before I added apps access was referencing single-port mode so I kept with that line.

Reply:
I think it would be great to add a sentence that this allows you to use CLIs for HTTP applications managed via Application Access without needing to log in separately for each command (if I'm understanding this feature correctly).

I also think we should confirm whether `tsh proxy app` requires TLS routing to work.

Reply:
> . While I'd like to document the addition I made, I think a potential followup issue might be best for clarifying the single-port mode comments that were originally made as well as documenting the other commands.

The other `tsh proxy` subcommands are already documented for example[ tsh proxy db](https://goteleport.com/docs/database-access/guides/gui-clients/#get-connection-information) and mared that this shoudl be used in `TLS routing` proxy mode.

Reply:
> The other tsh proxy subcommands are already documented for example[ tsh proxy db](https://goteleport.com/docs/database-access/guides/gui-clients/#get-connection-information)

I was referring to the lack of reference within the [tsh cli reference](https://goteleport.com/docs/setup/reference/cli/#tsh)